io/io/test/RRawFile.cxx

```diff
@@ -12,6 +12,7 @@
 #include <utility>
 
 #include "gtest/gtest.h"
+#include "gmock/gmock.h"
```

Comment:
Is this actually used somewhere?

Reply:
Good catch, this is a remnant of when I was checking `io_uring` error messages.